docs/docs/architecture.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Architecture
+---
+# Eraser architecture
```

Comment:
title is already a top level title here

Reply:
What do you mean?

Reply:
I mean Architecture will be a top level title on the page, we don't need to add `# Eraser architecture` again